README.md

```diff
@@ -1,162 +1,74 @@
 
-#### Git
-If you have cloned this project already then you can skip this!
-GitHub has excellent guides for [Windows][GitWin], [MacOSX][GitMacOSX] and [Linux][GitLinux].
-[GitWin]: http://help.github.com/win-set-up-git/
-[GitMacOSX]: http://help.github.com/mac-set-up-git/
-[GitLinux]: http://help.github.com/linux-set-up-git/
+###Clone the repo
+If you have cloned this project already then you can skip this, otherwise you'll need to clone this repo using Git. If you do not know how to clone a GitHub repository, check out this [help page] (https://help.github.com/articles/cloning-a-repository/) from GitHub.
 
-#### Python
-Before you will be able to run these tests you will need to have [Python 2.6][Python] installed.
-[Python]: http://www.python.org/download/releases/2.6.6/
+###Create or activate a Python virtual environment
+You should install this project's dependencies (which is described in the next step) into a virtual environment in order to avoid impacting the rest of your system, and to make problem solving easier. If you already have a virtual environment for these tests, then you should activate it, otherwise you should create a new one.
+For more information on working with virtual environments see our [our quickstart guide] (https://wiki.mozilla.org/QA/Execution/Web_Testing/Automation/Virtual_Environments) and also [this blog post] (http://www.silverwareconsulting.com/index.cfm/2012/7/24/Getting-Started-with-virtualenv-and-virtualenvwrapper-in-Python).
 
-Install pip (for managing Python packages):
+### Install dependencies
+Install the Python packages that are needed to run our tests using pip. In a terminal, from the the project root, issue the following command:
 
-    sudo easy_install pip
+    pip install -Ur requirements.txt
 
-#### Before you begin
-Before you dive into running tests you will want to register for an account on 
-[Addons-Dev][AddonsDev].  For security reasons you may want to create
-this account with a different username and password than you may have on the production addons site.
-[AddonsDev]: https://addons-dev.allizom.org/en-US/firefox/
+### Create a variables.json file
+Some of the tests in this repo need to log into the addons website and/or PayPal. We store the credentials (i.e., username and password) to access those sites in a file called `variables.json`, which we then pass to the tests via the command line. If you want to be able to run any of those tests, you will need your own copy of the `variables.json` file, which you will update to contain your own credentials.
+To do that, make a copy of the `variables.json` file which exists in the project root and update that with your own credentials. You will then pass the name of that file on the command line. For the purposes of the examples below, assume you named your copy of the file `my_variables.json`.
```

Comment:
Is this enough information? It doesn't cover how to determine the values for the file, which I suspect is one of the main barriers.